src/components/datatable/DocumentsTable.vue

```diff
@@ -46,6 +48,25 @@
     return Object.assign(result, options);
   }
 
+  // comparator functions for custom sort of "complicated" columns
+  // these functions could be removed if the sort function of
+  // AG grid could be deactivated since documents are expected to
+  // be given in correct order by the backend
+  function condComp(cond1, cond2) {
+    const conds = { 'excellent': 0, 'good': 1, 'average': 2, 'poor': 3, 'awful': 4 };
+    return conds[cond1] - conds[cond2];
+  }
+
+  function qualComp(qual1, qual2) {
+    const quals = { 'empty': 0, 'draft': 1, 'medium': 2, 'fine': 3, 'great': 4 };
+    return quals[qual1] - quals[qual2];
+  }
+
+  function avComp(av1, av2, n1, n2, isInverted) {
+    const avs = { 'level_ns': 0, 'level_1': 1, 'level_2': 2, 'level_3': 3, 'level_4': 4, 'level_5': 5 };
```

Comment:
Is it `level_ns` or `level_na` ? (if first, good argument for using array for common.js)